tests/wp-unit/tests/include/Core/Pages/Metabox/DeletePagesByStatusMetaboxTest.php

```diff
@@ -26,20 +26,24 @@ public function setUp() {
 		$this->metabox = new DeletePagesByStatusMetabox();
 	}
 
+	/**
+	 * Test that pages from a single post status (published pages) can be trashed.
+	 */
 	public function test_that_published_pages_can_be_trashed() {
 		$this->factory->post->create_many( 10, array(
 			'post_type' => 'page',
 		) );
 
-		$published_page = $this->get_pages_by_status();
-		$this->assertEquals( 10, count( $published_page ) );
+		$published_pages = $this->get_pages_by_status();
+		$this->assertEquals( 10, count( $published_pages ) );
 
 		$delete_options = array(
 			'publish'      => 'published_pages',
 			'drafts'       => '',
 			'pending'      => '',
 			'future'       => '',
 			'private'      => '',
+			'post_status'  => 'publish',
```

Comment:
You should not be passing `post_status` in `delete_options`.

This is the configuration that is got by parsing the user request.

You need to remove this for all test cases.